forest/benchmarking/tests/test_state_tomography.py

```diff
@@ -163,7 +168,7 @@ def two_q_tomo_fixture():
 
     # Get data from QVM
     tomo_expt = generate_state_tomography_experiment(state_prep, qubits)
-    results = list(measure_observables(qc=qc, tomo_experiment=tomo_expt, n_shots=100_000))
+    results = list(measure_observables(qc=qc, tomo_experiment=tomo_expt, n_shots=4000))
```

Comment:
Will this change require changing the tolerances in the tests that depend on these results?

Reply:
No, the tests all pass still. It's just that one SCS issue with the diamond norm stuff.